server/app.ts

```diff
@@ -42,6 +43,7 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpAuthentication())
   app.use(authorisationMiddleware())
   app.use(setUpCsrf())
+  app.use(populateValidationErrors())
```

Comment:
unimportant: i think this can go after the prisoner image route, we don't need this middleware for that route.

Reply:
Done :)